### PR TITLE
H-3071: Fix hashdotdesign vercel install step

### PR DESCRIPTION
**Pull request by TimDiekmann**
## 🌟 What is the purpose of this PR?

`hashdotdesign` currently fails to deploy

### CODE REVIEW

**Review by CiaranMn — APPROVED** [review]

**Review by vilkinsons — APPROVED** [review]

**Comment by TimDiekmann** [comment]
It's failing on https://github.com/hashintel/hash/pull/4688 so I want to wait until that is merged before merging this.

**Pull request merged**
